src/sass/_custom_select.scss

```diff
@@ -0,0 +1,76 @@
+.p-custom-select {
+  // style copied directly from vanilla-framework for the select element
+  .p-custom-select__toggle {
+    @include vf-icon-chevron-themed;
+    @extend %vf-input-elements;
+
+    // stylelint-disable property-no-vendor-prefix
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    // stylelint-enable property-no-vendor-prefix
+    background-position: right calc(map-get($grid-margin-widths, default) / 2)
+      center;
+    background-repeat: no-repeat;
+    background-size: map-get($icon-sizes, default);
+    box-shadow: none;
+    min-height: map-get($line-heights, default-text);
+    padding-right: calc($default-icon-size + 2 * $sph--small);
+    text-indent: 0.01px;
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    // this emulates the highlight effect when the select is focused
+    // without crowding the content with a border
+    &.active {
+      box-shadow: inset 0 0 0 3px $color-focus;
+    }
+  }
+
+  // custom style introduced by this project
+  .p-custom-select__label {
```

Comment:
The name here is confusing. This is about the toggleLabel, which is the text in the toggle. Not the label of the custom select itself.